pelican/generators.py

```diff
@@ -727,8 +727,13 @@ def generate_output(self, writer):
             source_path = os.path.join(self.path, sc.source_path)
             save_as = os.path.join(self.output_path, sc.save_as)
             mkdir_p(os.path.dirname(save_as))
-            shutil.copy2(source_path, save_as)
             logger.info('Copying %s to %s', sc.source_path, sc.save_as)
+            try:
+                # workaround for Android python copy2 bug ([issue28141])
```

Comment:
I think it might be clearer to link to the issue page here, as `[issue28141]` feels a little ambiguous.